content-src/components/LinkMenuButton/LinkMenuButton.scss

```diff
@@ -18,4 +18,10 @@
   transition-property: transform, opacity;
   transition-duration: 200ms;
   z-index: 399;
+
+  &:focus,
+  &:active {
+    transform: scale(1);
```

Comment:
when the link-menu-button is activated through tabbing it disappears without this little bit of css